### PR TITLE
feat: apply feedback

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [x] Not that difficult
- [ ] Approved feature branch merge to master

### Description
SSIA

### CODE REVIEW

**Review by piggggggggy — APPROVED** [review]
LGTM~

File: apps/web/src/lib/site-initializer/last-accessed-workspace.ts [hunk 1]
```diff
@@ -20,10 +21,13 @@ export const setCurrentAccessedWorkspaceId = async (workspaceId?: string): Promi
...
 export const getLastAccessedWorkspaceId = async (): Promise<string | undefined> => {
     try {
-        const response = await SpaceConnector.clientV2.config.userConfig.get<UserConfigGetParameters, UserConfigModel>({
+        const { results } = await SpaceConnector.clientV2.config.userConfig.list<UserConfigListParameters, ListResponse<UserConfigModel>>({
             name: LAST_ACCESSED_WORKSPACE_KEY,
         });
-        return response.data.workspace_id;
+        if (!results) {
+            return undefined;
+        }
```
Comment (yuda110) [inline apps/web/src/lib/site-initializer/last-accessed-workspace.ts:29]: todo: 
```typescript
if (!results?.length)
```

File: apps/web/src/services/asset-inventory/components/CloudServiceTagsPanel.vue [hunk 2]
```diff
@@ -145,11 +143,11 @@ watch([() => props.resourceId, () => props.resourceId], () => { getCloudServiceT
...
         </template>
         <template #col-provider-format="{ value }">
-            <p-badge v-if="value"
-                     :background-color="getProviderBadgeOption(value).color"
+            <p-badge v-if="storeState.providers[value]"
+                     :background-color="storeState.providers[value].color"
                      text-color="white"
             >
-                {{ getProviderBadgeOption(value)?.label }}
+                {{ storeState.providers[value].label }}
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/components/CloudServiceTagsPanel.vue:150]: nitpick: optional chaining here!

File: apps/web/src/services/asset-inventory/pages/CloudServiceDetailPage.vue [hunk 3]
```diff
@@ -547,6 +552,14 @@ debouncedWatch([() => props.group, () => props.name, () => props.provider], asyn
                                 />
                             </p-text-button>
                         </template>
+                        <template #col-provider-format="{value}">
+                            <p-badge v-if="storeState.providers[value]"
+                                     :background-color="storeState.providers[value].color"
+                                     text-color="white"
+                            >
+                                {{ storeState.providers[value].label }}
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/pages/CloudServiceDetailPage.vue:560]: nitpick: here, too!

**Review by yuda110 — APPROVED** [review]
LGTM~!!!

**Pull request merged**